tox.ini

```diff
@@ -56,10 +57,15 @@ commands_pre =
     # TODO: move this to more generic deps once stable 2.8.0 is out
     ansible28: pip install --pre 'ansible >= 2.8b1,<2.9'
     find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -path '*/__pycache__/*' -name '*.py[c|o]' -delete
+    functional: sh -c 'export HCLOUD_TOKEN=$(./contrib/tokenmgmr get -d hetznercloud)'
```

Comment:
I'm pretty sure this won't work: you invoke sh subshell which will probably lose env vars in its internal session.
It looks like this should probably be done on the CI level.